resources/main.css

```diff
@@ -48,6 +48,9 @@
 	color: red;
 	font-weight: bold;
 }
-.mw-scratch-confirmaccount-click-copy {
+#mw-scratch-confirmaccount-click-copy {
```

Comment:
Undo this in conjunction with my other comment please.

Reply:
ok